pkg/pgclient/client.go

```diff
@@ -22,10 +23,22 @@ import (
 	"github.com/timescale/promscale/pkg/prompb"
 	"github.com/timescale/promscale/pkg/promql"
 	"github.com/timescale/promscale/pkg/query"
+	"github.com/timescale/promscale/pkg/telemetry"
 	"github.com/timescale/promscale/pkg/tenancy"
 	"go.opentelemetry.io/collector/model/pdata"
 )
 
+var PromscaleID uuid.UUID
+
+func init() {
+	// PromscaleID must always be generated on start, so that it remains constant throughout the lifecycle.
+	PromscaleID = uuid.New()
+}
+
+// Post connect validation function, useful for things such as acquiring locks
+// that should live the duration of the connection
```

Comment:
Lets write the comment in the correct style (start with name of type alias and write complete sentences).